specs/indexOf.test.ts

```diff
@@ -6,16 +6,14 @@ describe('indexOf', () => {
   it("should return a number indicating an object's position in a list", () => {
     const list = [0, 10, 20, 30]
     eq(indexOf(30, list), 3)
-  })
-
-  it('should return -1 if the object is not in the list', () => {
-    const list = [0, 10, 20, 30]
     eq(indexOf(40, list), -1)
+    eq(indexOf(0, list), 0)
   })
 
-  const input = [1, 2, 3, 4, 5]
+  const input = [1, 2, 1, 2, 5]
   it('should return the index of the first item', () => {
```

Comment:
rename test to `should return the index of the first item when there are duplicates of such item`